rollouts/manifests/crds/containercluster.yaml

```diff
@@ -0,0 +1,1351 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    cnrm.cloud.google.com/version: 1.100.0
+    kubectl.kubernetes.io/last-applied-configuration: |
```

Comment:
Is this CRD supposed to have things like annotations (last-applied-configuration), generation, resourceVersion, etc?

Reply:
No, I extracted this from a cluster and forgot to clean it up. Removed runtime info from it.